test/shared_array_scope_list.rb

```diff
@@ -60,6 +60,11 @@ def test_insert
       assert !new.first?
       assert new.last?
 
+      new = ArrayScopeListMixin.act_as_list_no_update do
+        ArrayScopeListMixin.create(parent_id: 20, parent_type: 'ParentClass')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.